src/triage.md

```diff
@@ -0,0 +1,47 @@
+# 🔍 Triage meetings
+
+## When, where
+
+The weekly triage meeting is held on [Zulip] according with the calendar available at [rust-lang/calendar].
```

Comment:
It might be good to have the ICS links here too, or some other way to make it easy to see what the schedule is. Or even saying something like "meetings currently are held on Fridays at 16:00 UTC, but see the Rust calendar for the most up to date information, and watch #wg-macros for last minute changes."

Reply:
>meetings currently are held on Fridays at 16:00 UTC, but see the Rust calendar for the most up to date information, and watch #wg-macros for last minute changes.

I like this toos